di-ipv-orchestrator-stub/src/main/java/uk/gov/di/ipv/stub/orc/utils/JwtBuilder.java

```diff
@@ -72,6 +73,7 @@ public static JWTClaimsSet buildAuthorizationRequestClaims(
                 .claim("govuk_signin_journey_id", signInJourneyId)
                 .claim("persistent_session_id", UUID.randomUUID().toString())
                 .claim("email_address", "dev-platform-testing@digital.cabinet-office.gov.uk")
+                .claim("vtr", List.of("Cl.Cm.P2", "Cl.Cm.PCL200"))
```

Comment:
This corresponds to a specific HMRC inherited identity - I think the default value should be `Cl.Cm.P2`, with an option to override it with something else.

(That option could also be left until a later ticket, since we won't need it until inherited identities comes along)

Reply:
Thanks Joe ofr your comment and informations and raised a new PR what The 'vtr' value is now being retrieved with a default value on the front end. It can be changed if desired. https://github.com/govuk-one-login/ipv-stubs/pull/570